mobile-app/Screens/HomeScreen.js

```diff
@@ -45,47 +45,47 @@ export function HomeScreen({ navigation }) {
                 <ImageBackground source={require('../assets/Background.png')} resizeMode={"stretch"} style={styles.image}>
                     <View style={{flex: 0.3}} />
                     <View style={styles.flexBox}>
-                        <TouchableOpacity style={{flexDirection: 'column'}} onPress={() => navigation.navigate('UserMainInterface', employees[0])}>
+                        <TouchableOpacity style={styles.userBox} onPress={() => navigation.navigate('UserMainInterface', employees[0])}>
                             <View style={styles.square}>
                                 <Image style={styles.image} source={{uri: employees[0]["avatarUrl"]}}/>
                             </View>
                             <View style={styles.rectangle}>
                                 <Text style={styles.interfaceIconsText}>{employees[0]["nickName"]}</Text>
                             </View>
                         </TouchableOpacity>
-                        <TouchableOpacity style={{flexDirection: 'column'}} onPress={() => navigation.navigate('UserMainInterface', employees[1])}>
+                        <TouchableOpacity style={styles.userBox} onPress={() => navigation.navigate('UserMainInterface', employees[1])}>
                             <View style={styles.square}>
                                 <Image style={styles.image} source={{uri: employees[1]["avatarUrl"]}}/>
                             </View>
                             <View style={styles.rectangle}>
                                 <Text style={styles.interfaceIconsText}>{employees[1]["nickName"]}</Text>
                             </View>
                         </TouchableOpacity>
-                        <TouchableOpacity style={{flexDirection: 'column'}} onPress={() => navigation.navigate('UserMainInterface', employees[2])}>
+                        <TouchableOpacity style={styles.userBox} onPress={() => navigation.navigate('UserMainInterface', employees[2])}>
                             <View style={styles.square}>
                                 <Image style={styles.image} source={{uri: employees[2]["avatarUrl"]}}/>
                             </View>
                             <View style={styles.rectangle}>
                                 <Text style={styles.interfaceIconsText}>{employees[2]["nickName"]}</Text>
                             </View>
                         </TouchableOpacity>
```

Comment:
Wszystkie takie wystąpienia można by było na luziku zamknąć w pętli:
`{ employees.map((employee) => {
return <TouchableOpacity style={styles.userBox} onPress={() => navigation.navigate('UserMainInterface', employee)}>
                            <View style={styles.square}>
                                <Image style={styles.image} source={{uri: employee.avatarUrl}}/>
                            </View>
                            <View style={styles.rectangle}>
                                <Text style={styles.interfaceIconsText}>{employee.nickName}</Text>
                            </View>
                        </TouchableOpacity>;
}
) }`
czy coś takiego 😄

Reply:
wiem i kusilo mnie zeby takz robic ale ekran ten robila alicja ja tylko dalem api z bazy danych naprawde nie lubie frontu xd

Reply:
hahah, rozumiem :D ja tylko komentuje jak widzę, żebyście przy okazji nauczyli się też dobrych praktyk. 
@Generacja @XsedoX Właśnie odpaliłam sobie tą apkę, super to wygląda, dzięki!